product_docs/docs/pgd/4/rel_notes/pgd_4.3.2_rel_notes.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.2"
+navTitle: "Version 4.3.2"
+---
+
+EDB Postgres Distributed version 4.3.2 is a minor release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8 and later.
+
+| Component | Version |  Type           |                                                   Description                                                           |
+| --------- | ------- | --------------- | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.3.2   | Bug fix         | Fixed a bug in handling of memory context in presence of triggers. |
+| BDR       | 4.3.2   | Bug fix         | Prevent overflow of EagerTxnConfig::node_idss array |
+| BDR       | 4.3.2   | Bug fix         | Fixed `bdr.sequence_alloc` upgrade script. |
+| BDR       | 4.3.2   | Bug fix         | Fixed check for malformed connection string tp prevent failure in `bdr.create_node()`. (RT95453) |
+| BDR       | 4.3.2   | Bug fix         | Remove txn_config entry from ReorderBuffer hash table. |
+| BDR       | 4.3.2   | Bug fix         | Fixed a memory leak in `bdr.sequence_alloc`. |
+| BDR       | 4.3.2   | Bug fix         | Ignore global_lock check from repset_func when SDW enabled |
+| BDR       | 4.3.2   | Bug fix         | Added check for conflicting node names. |
+| BDR       | 4.3.2   | Bug fix         | Added wrapper for checking RTE permissions to disable pgaudit |
+| BDR       | 4.3.2   | Bug fix         | Fixed the consensus snapshot compatibility with PGD 3.7. (RT93022) |
+| BDR       | 4.3.2   | Bug fix         | Fixed an issue by allowing a logical join of node if there are foreign key constraints violations. (RT91745) |
```

Comment:
"Don't check foreign keys during node join, this makes the join behave the same as replication"